ext/src/main/java/org/jboss/logmanager/ext/handlers/PeriodicRotatingFileHandler.java

```diff
@@ -44,6 +47,13 @@
     private TimeZone timeZone = TimeZone.getDefault();
     private SuffixRotator suffixRotator = SuffixRotator.EMPTY;
 
+    private enum PruningStrategy { NONE, PERIODS, SIZE };
```

Comment:
When I think about the other options at a configuration level that are current available, I am really mostly attracted to leaving the PruningStrategy.PERIODS option, but this is great!